docs/config.json

```diff
@@ -597,6 +597,10 @@
           "title": "Using tsh",
           "slug": "/connect-your-client/tsh/"
         },
+        {
+          "title": "Using the Web UI",
```

Comment:
Because this page is only about joining sessions in the Web UI, this title is misleading. Most of the other features of the Web UI are covered in the Introduction page of this section, under the "Web UI" Tabs.

Reply:
This was Paul's suggestion - right now this is the only thing on the page, but we will add more information on the web UI here.

Reply:
If it makes sense, we could plan to add instructions for potentially confusing/hard-to-find Web UI operations to this page within the coming weeks.

Reply:
Some docs is better than no docs. I don't think we should hold up valuable info from our docs thats been missing for years just because there's still more that can be documented (that will always be the case). Can we get this merged and continue to iterate?